src/Platform.c

```diff
@@ -464,24 +464,162 @@ void PltWaitForConditionVariable(PLT_COND* cond, PLT_MUTEX* mutex) {
 #endif
 }
 
-uint64_t PltGetMillis(void) {
+//// Begin SDL time emulation
+
+// These functions mostly emulate SDL_GetTicks64(), because this library doesn't
+// use SDL, but timing data needs to be compatible with SDL-supplied data in moonlight-qt.
```

Comment:
I'm not really sure I understand the motivation of trying to match SDL. The epoch of `PltGetMillis()` has always been implementation-defined, and the app can normalize it in whatever way it chooses by calling `PltGetMillis()` .

We can't promise SDL-style times, because we have no control over when the client chooses to call `LiStartConnection()`. It's quite likely that the SDL timebase and our timebase will differ significantly, since successful network requests to the host need to happen before `LiStartConnection()` can be called to begin negotiating the connection.

Reply:
I remember why I said this: it's because there are a lot of calls in -qt to `SDL_GetTicks()` for tracking time, as well as calls to `LiGetMillis()`. Since they both offer millisecond-resolution it doesn't really matter which is used as long as they aren't compared with each other for timing purposes. I don't think they are ever mixed, but the potential for confusion is there. I wanted to replace both calls with a microsecond version, so when it came to writing the Li version it made sense to me to just use the same "counter starts at first API call" plan that SDL uses. I'd suggest that we go with the version in my patch because it's well-tested code from SDL. And then we remove my reference to SDL as well as the start time of the program. How about this wording for the 2 functions in Limelight.h?
```
// This function returns a time in microseconds with an implementation-defined epoch. It should only ever be compared with the return value from a previous call to itself.
```